napalm/nxos_ssh/nxos_ssh.py

```diff
@@ -484,7 +483,7 @@ def parse_uptime(uptime_str):
         Return the uptime in seconds as an integer
         """
         # Initialize to zero
-        (years, weeks, days, hours, minutes) = (0, 0, 0, 0, 0)
+        (years, weeks, days, hours, minutes, seconds) = (0, 0, 0, 0, 0, 0)
```

Comment:
Since this is a staticmethod, can we add a new file `test/nxos_ssh/test_parsers.py` that has unit tests for this function?

You can see `test/eos/test_versions.py` as an example of some simple tests.

Reply:
Done!